aries-site/src/examples/components/layer/LayerCenterExample.js

```diff
@@ -25,18 +33,21 @@ export const LayerCenterExample = () => {
                 <Box justify="center">
                   <MailOption />
```

Comment:
Can we confirm the icon being used here? The one in Figma seems rounded which is the aesthetic to which we are headed. It would be good for us to be aligned.

May require follow on issues.

Reply:
I may have to follow up because I looked through our icons and I do not see this one

Reply:
<img width="894" alt="Screen Shot 2022-02-25 at 3 09 30 PM" src="https://user-images.githubusercontent.com/42451602/155809840-8b87b647-4a9f-41cc-9dfe-017e7601481c.png">
I dont see a rounded email in our icons 😕

Reply:
made a new ticket to look into icon
https://github.com/grommet/hpe-design-system/issues/2400